e2e/terraform/provision-infra/network.tf

```diff
@@ -89,6 +89,17 @@ resource "aws_security_group" "servers" {
   }
 }
 
+# Allows Consul network access to Nomad, which is necessary when creating
+# auth methods. Must be done in an aws_security_group_rule to avoid cycles.
+resource "aws_security_group_rule" "consul_ingress" {
+  type                     = "ingress"
+  from_port                = 0
+  to_port                  = 0
+  protocol                 = "-1"
+  security_group_id        = aws_security_group.servers.id
+  source_security_group_id = aws_security_group.consul_server.id
+}
```

Comment:
This should probably just be an `ingress` rule on `resource.aws_security_group.servers` so that all the rules are defined in one place.

Reply:
Oh, of course!